### ISSUE TITLE
What's the meaning of entropy_loss in your cl_train.py?

### ISSUE THREAD

**Issue by donghaoye** [Tushar-N/blockdrop#2]

**Comment by zxwu** [comment]
It is used to encourage the prob distribution to be spiky as a way to encourage exploration.

**Comment by donghaoye** [comment]
I suggest that "entropy_loss = -probs*torch.log(probs)" and "loss = (loss - entropy_loss)/input_var.size(0)" in your code, which don't lead to misunderstand.
